frontend/public/components/RBAC/bindings.jsx

```diff
@@ -430,21 +430,30 @@ const Section = ({ label, children }) => (
   </div>
 );
 
-class BaseEditRoleBindingWithTranslation extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.subjectIndex = props.subjectIndex || 0;
-
-    const existingData = _.pick(props.obj, [
-      'metadata.name',
-      'metadata.namespace',
-      'roleRef',
-      'subjects',
-    ]);
-    existingData.kind = props.kind;
-    const { subjectKind, subjectName } = this.props.fixed.subjectRef || {};
-    const data = _.defaultsDeep({}, props.fixed, existingData, {
+const BaseEditRoleBindingWithTranslation = (props) => {
```

Comment:
`WithTranslation` is no longer needed now that this is a function component and you are using the hook below.  Can rename to `BaseEditRoleBinding` and remove `const BaseEditRoleBinding = withTranslation()(BaseEditRoleBindingWithTranslation);` on line 745.